src/applications/personalization/profile-2/components/Profile2Wrapper.jsx

```diff
@@ -48,7 +50,7 @@ const Profile2 = ({ children, routes, isLOA3, isInMVI }) => {
         </Breadcrumbs>
       </div>
 
-      <ProfileHeader />
+      {isEmpty(hero.errors) && <ProfileHeader />}
```

Comment:
Could you also update the Profile2Wrapper tests to add cases for when it should show the header and when it shouldn't?

Reply:
Done! 👍